lib/services/remove_from_cart.rb

```diff
@@ -0,0 +1,27 @@
+require_relative './fetch_product_from_storage'
+
+module Store
+  class RemoveFromCart
+    def initialize(params)
+      @product_id = params.fetch('product_id').to_i
+    end
+
+    def call
+      item = FetchProductFromStorage.new.call(@product_id, CART)
+
+      return unless item
+
+      remove_item(item)
```

Comment:
or I would put `remove_item` body to `call` and combine `if` condition with both, `item` and `item.quantity`